install_heroku_script.sh

```diff
@@ -0,0 +1,9 @@
+cd iframely/
+npm install
+echo "---------------1-------------------"
+cd ../
+cd oauth-proxy/
+npm install
+echo "------------------------------2--------------------------"
+cd ../
+gulp dev
```

Comment:
@sudheesh001 could you please elaborate a bit more. I am not sure if I follow you here!